test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,60 @@
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+const md5 = require('md5');
+
+chai.use(chaiSubset);
+
+const urlrepos = 'https://api.github.com/repos/aperdomob/jasmine-json-report';
+
+describe('Repositories GET', () => {
+  let userResponse;
+
+  before(async () => {
+    userResponse = await axios.get('https://api.github.com/users/aperdomob');
+  });
+  it('Consume GET Service on GITHUB API', () => {
+    expect(userResponse.status).to.equal(StatusCodes.OK);
+    expect(userResponse.data.name).to.equal('Alejandro Perdomo');
+    expect(userResponse.data.company).to.equal('Perficient Latam');
+    expect(userResponse.data.location).to.equal('Colombia');
+  });
+
+  describe('Using hypermedia', () => {
+    let jasmineRepo;
+    before(async () => {
+      const repositoriesResponse = await axios.get(`${userResponse.repos_url}`);
```

Comment:
each time you use userResponse, you need to call the data, since we need to access to the response body. So call to the get should be `userResponse.data.repos_url`

Reply:
yes!!!!!
Thank you!!!
It means that I got confused tryong to apply your review, I'm sorry, thanks for your patience!